ARP.Authentication/LdapHelper.cs

```diff
@@ -30,42 +30,73 @@ namespace garb.Authentication
 	public class LdapHelper
 	{
 		public static string LdapDomain;
+        public static string LocalAdminName;
+        public static string LocalAdminPwd;
+        public static string LocalReadOnlyName;
+        public static string LocalReadOnlyPwd;
 
-		public static bool ValidateDomainCredentials(UnitOfWork _unitOfWork, string username, string password)
+        public static bool ValidateDomainCredentials(UnitOfWork _unitOfWork, string username, string password)
 		{
-			//string domainName =  ;
-			string userDn = $"{username}@{LdapDomain}";
-
-			try
-			{
-				// Using Novell LdapConnection instead of unsupported System.Directory services
-				using (var connection = new LdapConnection { SecureSocketLayer = false })
-				{
-					connection.Connect(LdapDomain, LdapConnection.DEFAULT_PORT);
-					connection.Bind(userDn, password);
-
-					if (connection.Bound)
-					{
-						var userRepo = _unitOfWork.UserRepository;
-
-						var user = userRepo.GetByID(username);
-
-						if (user == null)
-						{
-							User newUser = new User() { UserName = username };
-							userRepo.Insert(newUser);
-							_unitOfWork.Save();
-						}
-
-						return true;
-					}
-				}
-			}
-			finally
-			{
-			}
-
-			return false;
-		}
-	}
+            if (username.Equals(LocalAdminName) && password.Equals(LocalAdminPwd) ||
```

Comment:
Please check first if those values are not null or empty - when they are not set they should not allow anyone to login with empty creds.

Reply:
Changed LdapHelper.cs to check for null or empty username and password.